include/aws/cal/symmetric_cipher.h

```diff
@@ -236,6 +238,12 @@ AWS_CAL_API struct aws_byte_cursor aws_symmetric_cipher_get_key(const struct aws
  */
 AWS_CAL_API bool aws_symmetric_cipher_is_good(const struct aws_symmetric_cipher *cipher);
 
+/**
+ * Retuns the current state of the cipher. If the cipher is finished that means it must
```

Comment:
nit: would rephrase it to say that ready is the state where cipher is usable and error/finished require reset before using again